ddl/table.go

```diff
@@ -131,9 +132,9 @@ func (d *ddl) onDropTable(t *meta.Meta, job *model.Job) error {
 			break
 		}
 		// finish this job
-		job.Args = []interface{}{tblInfo}
 		job.State = model.JobDone
 		job.SchemaState = model.StateNone
+		addFinishInfo(job, ver, nil)
```

Comment:
Because only alter table operations need to add tblInfo, you can read issue #1842.